page-template-cluster.php

```diff
@@ -147,6 +139,50 @@
 	</section>
 	<?php endif; ?>
 	<!-- End Research -->
+	<!-- Start Academics -->
+	<?php if ( is_array( $cluster_programs ) && ! empty( $cluster_programs ) ) : ?>
+	<section class="jumbotron bg-inverse text-inverse" aria-labelledby="research-programs">
+		<div class="container">
+			<div class="row">
+				<div class="col-lg-8">
+					<h2 id="research-programs" class="h1 mb-4 text-primary font-weight-black section-heading"><?php echo $post->post_title; ?> Degree Programs</h2>
+					<?php if ( $cluster_prg_copy ) : ?>
+					<div class="mb-5">
+						<p><?php echo $cluster_prg_copy; ?></p>
```

Comment:
To support multiple paragraphs in the program lead copy, consider removing this wrapper `<p>` tag and update the Program Lead Copy field to use a WYWISYG instead of a plain textfield.

Reply:
I don't think we want to support multiple paragraphs here.

Reply:
Updated the field description and added a character limit.